internal/build/docker.go

```diff
@@ -145,18 +147,18 @@ func (l *localDockerBuilder) PushDocker(ctx context.Context, ref reference.Named
 	return pushedDigest, nil
 }
 
-func (l *localDockerBuilder) buildBaseWithMounts(ctx context.Context, baseDockerfile string, mounts []model.Mount) (digest.Digest, error) {
-	err := validateDockerfile(baseDockerfile)
+func (l *localDockerBuilder) buildFromDfWithFiles(ctx context.Context, df string, paths []pathMapping) (digest.Digest, error) {
```

Comment:
i feel like this `df` string is crying out for a more structured object, even if it's something dumb like

```
type Dockerfile string

func (f Dockerfile) Plus(cmd string) Dockerfile {
  return Dockerfile(fmt.Sprintf("%s\n%s", f, cmd))
}
```

Reply:
good idea -- @jazzdan is on it!